src/newrelic.coffee

```diff
@@ -27,7 +27,7 @@
 #   hubot newrelic users emails - Returns a list of all user emails
 #   hubot newrelic alerts - Returns a list of active alert violations matching the current channel's subscriptions
 #   hubot newrelic alerts all - Returns a list of all active alerts
-#   hubot newrelic alerts subscribe <pattern> subscribe the current channel to alerts matching <pattern>
+#   hubot newrelic alerts subscribe <pattern> subscribe the current channel to alerts whose policy name(s) matches <pattern>. Regexes are OK, eg cf.gov*
```

Comment:
This might steer people wrong--  these aren't regexes, more like shell glob patterns https://github.com/isaacs/minimatch

Reply:
Great point. How about a phrase like "Simple matching patterns such as '*' will work" ? Realistically, that's probably the 99.99% use case

Reply:
That should be fine

Reply:
Done.